packages/bpk-component-banner-alert/package-lock.json

```diff
@@ -1,17 +1,153 @@
 {
-	"requires": true,
+	"name": "bpk-component-banner-alert",
```

Comment:
There should be no change to this file as you haven't modified `package.json`. Can you perhaps attempt to diagnose why this happens?

cc @matteo-hertel i've seen this in other MR's and I suspect that it'll come up as part of the node 8 / npm5 migration that we'll need to have an explanation / workaroudn for... 🤔

Reply:
Very good point, let me investigate

Reply:
see https://github.com/Skyscanner/backpack/pull/247#discussion_r149629803 not sure what's causing it tough